### DISCUSSION TITLE
Using the intercept with graphQL and Browserstack tunnels with Jenkins

### DISCUSSION THREAD

**Discussion by NutriMarc** [webdriverio-community/wdio-intercept-service#666] [Q&A]
Tech Stack
- Jenkins executer
- Browserstack local tunnel(JenkinsInstance <--> BrowserstackDevice)
- WDIO test Runner
- Intercept Service
- GraphQL

We are currently trying to use the intercept service to capture/return graphQL requests/responses. Has anyone used the intercept service with the tech stack mentioned? I can get more information but felt the first step was in the problem statement. From there it would be nice to see some examples. 

Does anyone have any experience with this?

**Comment by rohit-y180** [comment]
Are you able to achieve the use-case when you are running this use-case on your local machine/device instead of running the test on BrowserStack devices?

**Reply by NutriMarc** [reply]
Yes
